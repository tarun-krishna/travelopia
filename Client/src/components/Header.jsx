import React from "react";
import car from "../images/car.png";
import "../styles/header.css";
const Header = ({ show, setShow }) => {
  // Header Component
  return (
    <div className="header-container">
      <div className="logo-title-container">
        <img className="car-logo" src={car} alt="car-logo" />
        <span className="header-title">Travelopia</span>
      </div>
      <div>
        <button className="show-user-details" onClick={() => setShow(!show)}>
          {show ? "Show Traveller's Details" : "Add Traveller"}
        </button>
      </div>
    </div>
  );
};

export default Header;
