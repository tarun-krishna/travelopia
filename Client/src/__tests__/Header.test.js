import Header from "../components/Header";
import renderer from "react-test-renderer";

describe("Header should render correctly", () => {
  it("renders correctly", () => {
    const header = renderer.create(<Header />).toJSON();
    expect(header).toMatchSnapshot();
  });
});
