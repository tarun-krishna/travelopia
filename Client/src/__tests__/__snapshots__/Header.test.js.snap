// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render correctly renders correctly 1`] = `
<div
  className="header-container"
>
  <div
    className="logo-title-container"
  >
    <img
      alt="car-logo"
      className="car-logo"
      src="car.png"
    />
    <span
      className="header-title"
    >
      Travelopia
    </span>
  </div>
  <div>
    <button
      className="show-user-details"
      onClick={[Function]}
    >
      Add Traveller
    </button>
  </div>
</div>
`;
